docs/src/api/params.md

```diff
@@ -405,7 +405,7 @@ An instance of [FormData] can be created via [`method: APIRequestContext.createF
 
 ## js-python-fetch-option-multipart
 * langs: js, python
-- `multipart` <[Object]<[string], [string]|[float]|[boolean]|[ReadStream]|[Object]>>
+- `multipart` <[FormData]|[Object]<[string], [string]|[float]|[boolean]|[ReadStream]|[Object]>>
```

Comment:
- Is `FormData` available in python?
- We will need a custom link on `playwright.dev` leading to MDN.

Reply:
> Is FormData available in python?

No, only csharp and java.

Reply:
https://github.com/microsoft/playwright.dev/pull/1360 adds FormData link